OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -198,6 +198,7 @@
     <Compile Include="Scripting\Global\FacingGlobal.cs" />
     <Compile Include="Scripting\Global\HSLColorGlobal.cs" />
     <Compile Include="Scripting\Global\UserInterfaceGlobal.cs" />
+    <Compile Include="Scripting\Properties\CaptureProperties.cs" />
```

Comment:
I suggest to move this into a separate pull request as it doesn't provide a test case yet and is independent from infiltrate.